src/locales/en/common.json

```diff
@@ -1758,10 +1766,34 @@
       }
     },
     "exchange": {
-      "title": "Buy / Sell"
+      "title": "Buy / Sell",
+      "providers": {
+        "title": "Choose a provider to buy crypto",
+        "whatIsSwap": "What is Buy?",
+        "cta": "Continue",
+        "coinify": {
+          "bullets": [
+            "Available in 130+ countries",
+            "EUR, USD, GBP and 32 others",
+            "SEPA & Faster Payments",
+            "Local US wire transfers",
+            "Credit and debit cards"
+          ]
+        },
+        "wyre": {
+          "bullets": [
+            "Only 🇺🇸 bank account",
```

Comment:
Do we really want to use emoji here?

Reply:
@rjarasson-ledger wdyt? ^^

Reply:
This is in the mockups, that's the wording that has been validated and revised multiple times.
I don't think anything about it.
It's working.

Reply:
I think it's ok. it's UTF-8.

Reply:
but maybe @juan-cortes mentions more the fact it's a "traductible". we could have injected by a `{flag}` but it's a bit overkill tbh. let's see how it appears on crowdin.

Reply:
using somthing like `{flag}` would imply logic on the caller. This is not possible since this is only content, with no feature linked to.
The component simply renders bullet point for each item in the array, we cannot do simpler that this.
Emoji is just a UI choice done before, and could be updated only through translation/content update.
Not feature here.

Reply:
fair enough.